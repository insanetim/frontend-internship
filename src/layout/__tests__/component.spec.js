import { shallow } from 'enzyme'
import Layout from '../component'

it('matches snapshot', () => {
  const component = shallow(<Layout />)

  expect(component).toMatchSnapshot()
})
