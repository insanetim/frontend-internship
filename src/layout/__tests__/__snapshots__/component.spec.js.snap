// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Layout>
  <Header />
  <Content>
    <Outlet />
  </Content>
  <ModalRoot />
</Layout>
`;
